tool/travis.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+# Fast fail the script on failures.		
+set -e
+
+# Verify that the libraries are error free.
+dartanalyzer --fatal-warnings \
+  bin/analyzer.dart
```

Comment:
you may want to add `lib/src/analyzer_impl.dart` here as well - dartanalyzer way ignore issues in code referenced via package: imports.